Softeq.XToolkit.Common/Collections/BiDictionary.cs

```diff
@@ -4,20 +4,14 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.IO;
-using System.Runtime.Serialization;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Security.Permissions;
 
 #pragma warning disable CS1591
 
 namespace Softeq.XToolkit.Common.Collections
 {
-    [Serializable]
```

Comment:
Why do we remove BiDictionary serialization?

Reply:
Because binary serialization is obsolete and disabled for .net8:
https://learn.microsoft.com/en-us/dotnet/core/compatibility/serialization/8.0/binaryformatter-disabled